src/tooltip.js

```diff
@@ -109,7 +109,8 @@
 
             trowEnter.append("td")
                 .classed("key",true)
-                .html(function(p, i) {return keyFormatter(p.key, i)});
+                .html(function(p, i) {return keyFormatter(p.key, i)})
+                .style("font-weight", function(p) {return p.total && "bold"});
```

Comment:
For something like this, it's better to have a class name and specify the syle elsewhere I think.